projects/m2/021-unique-characters/js/index.js

```diff
@@ -0,0 +1,28 @@
+// const prompt = require("prompt-sync")({ sigint: true}) ;
+
+function calculateUniqueCharacters(text){
+    let uniqueCharactersCount = 0;
+    const uniqueCharactersDictionary = [];
+
+    for (let i = 0; i < text.length; i++) {
+        const character = text.charAt(i);
+
+        if( ! uniqueCharactersDictionary.includes(character) ){        
+            uniqueCharactersDictionary.push(character);
+        } 
+    }

```

Comment:
Using a **set** instead of an array as the **data structure** might be interesting.
What do you think? Do you want to try it?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/Set

Reply:
Sure, thanks.